riscv/runtime/src/coprocessors.rs

```diff
@@ -4,9 +4,9 @@
 // The function itself will be removed by the compiler
 // during the reachability analysis.
 extern "C" {
-    fn poseidon_coprocessor(a: u32, b: u32) -> u32;
+    fn poseidon_coprocessor(a: u32, b: u32, c: u32) -> u32;
 }
 
 pub fn poseidon_hash(a: u32, b: u32) -> u32 {
-    unsafe { poseidon_coprocessor(a, b) }
+    unsafe { poseidon_coprocessor(a, b, 0) }
```

Comment:
the capacity is currently set to 0, which I think it wrong. Please advise.

Reply:
In my understanding, 0 is as good as any other constant value. However, the [Poseidon paper](https://eprint.iacr.org/2019/458.pdf) suggests some standard values in Section 4.2, depending on the use-case.